exercises/minesweeper/example.js

```diff
@@ -0,0 +1,46 @@
+var Minesweeper = function() {
+  this.distanceXdistanceYs = [
+      [-1,-1],
+      [-1,0],
+      [-1,1],
+
+      [1,1],
+      [1,0],
+      [1,-1],
+
+      [0,1],
+      [0,-1]
+
+  ]
+};
+
+
+Minesweeper.prototype.annotate = function(rows) {
+  if(rows.length === 0 || rows[0].length === 0) {
+    return rows;
+  }
+  var board = rows.map(function(row) { return row.split(""); });
+  var outBoard = [];
+  for(var x = 0; x < board.length; x++) {
+    outBoard[x] = [];
+    for(var y = 0; y < board[x].length; y++) {
+      var spot = board[x][y];
+      if(spot == "*") {
+        outBoard[x][y] = spot;
+        continue;
+      }
+      var bombCount = this.distanceXdistanceYs.map(function(dxdy) {
+        if(board[x + dxdy[0]] === undefined) return 0;

```

Comment:
Let's use curly braces here, even though they're not strictly required.  

And a space between `if` and `(` please.